mailman3_fedmsg_plugin.py

```diff
@@ -67,47 +78,44 @@ class Archiver(object):
         "user-agent",
     ]
 
-    def __init__(self):
-        if not getattr(getattr(fedmsg, '__local', None), '__context', None):
-            hostname = socket.gethostname().split('.')[0]
-            fedmsg.init(name="mailman.%s" % hostname)
-        self.config = fedmsg.config.load_config()
-
-
```

Comment:
Any reason why the class constructor was gotten rid of?

Reply:
In the original implementation of the Archiver class, the constructor was used to initialize the fedmsg library, load configuration, and perform other setup tasks.
I got rid of it because we wont be needing it, fedora messaging will start the connection, in the first Message.

Reply:
Do you suggest that it should remain?